### PR TITLE
feat: new design for OG user image

### LINKED ISSUE #1013: [Bug]: User OG image cannot render markdown
### What happened?

I guess we need to just render static text vs pull the beginning of the users bio. Its probably better that way because someone could put inappropriate text in their bio before we catch it

Design could be modified as well

CleanShot 2023-10-31 at 07 55 45@2x

- [ ] remove markdown for static text

### What browsers are you seeing the problem on?

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by Dun-sin**
## Description

- display the time since user joined in the OG
- change the display of the user OG image

## Related Issue

Closes #1013

## Screenshots/Video (if applicable):
image

### CODE REVIEW

File: apps/og-image/app/api/user/route.tsx [hunk 1]
```diff
@@ -21,21 +20,24 @@ export async function GET(req: Request) {
   }
 
   const props = parsed.data.input;
+  console.log({props})
```
Comment (Dun-sin) [inline apps/og-image/app/api/user/route.tsx:23]:
```suggestion
```

File: apps/web/src/app/[locale]/[username]/page.tsx [hunk 2]
```diff
@@ -54,16 +55,32 @@ export async function generateMetadata({ params: { username: usernameFromQuery }
...
+  const date = user?.createdAt as Date
+  const dateSince = getRelativeTime(date)
 
   if (username)
     return buildMetaForUser({
       username,
       title: `${username}'s profile | TypeHero`,
       description: `View the profile of ${username} on TypeHero.`,
-      bio,
+      challenges: challenges.length,
```
Comment (bautistaaa) [inline apps/web/src/app/[locale]/[username]/page.tsx:81]: we can remove the challenges code now i believe

Comment (Dun-sin) [reply apps/web/src/app/[locale]/[username]/page.tsx:81]: oh missed that
```suggestion
```

File: apps/web/src/app/[locale]/[username]/page.tsx [hunk 3]
```diff
@@ -54,16 +55,31 @@ export async function generateMetadata({ params: { username: usernameFromQuery }
...
+        some: {
+          userId: user?.id,
+          isSuccessful: true,
+        },
+      },
+    },
+    select: {
+      name: true,
+    },
+  });
```
Comment (Dun-sin) [inline apps/web/src/app/[locale]/[username]/page.tsx:70]:
```suggestion
```

File: apps/web/src/app/metadata.ts [hunk 4]
```diff
@@ -59,21 +59,24 @@ interface MetaParamsForUser {
...
-  bio,
+  challenges,
+  dateSince,
   avatar,
 }: MetaParamsForUser): Promise<Metadata> => {
   const params = `${userParam.toSearchString({
     username,
-    bio,
     avatar,
+    challenges,
```
Comment (Dun-sin) [inline apps/web/src/app/metadata.ts:78]:
```suggestion
```

File: apps/web/src/app/metadata.ts [hunk 5]
```diff
@@ -59,21 +59,24 @@ interface MetaParamsForUser {
...
+  dateSince: string;
+  challenges: number;
 }
 /** Helper to build opengraph metadata for a user, you should call this in generateMetadata() next function */
 export const buildMetaForUser = async ({
   title,
   description,
   username,
-  bio,
+  challenges,
```
Comment (Dun-sin) [inline apps/web/src/app/metadata.ts:71]:
```suggestion
```

File: apps/web/src/app/metadata.ts [hunk 6]
```diff
@@ -59,21 +59,24 @@ interface MetaParamsForUser {
   title: string;
   description: string;
   username: string;
-  bio: string;
   avatar: string;
+  dateSince: string;
+  challenges: number;
```
Comment (Dun-sin) [inline apps/web/src/app/metadata.ts:64]:
```suggestion
```

**Review by bautistaaa — APPROVED** [review]

**Comment by Dun-sin** [comment]
> can you run pnpm checks:fix. failing some quality checks in CI

will do

**Comment by Dun-sin** [comment]
@bautistaaa done

**Pull request merged**